appveyor.yml

```diff
@@ -8,8 +8,6 @@ image:
 - Visual Studio 2015
 - Visual Studio 2017
 - Visual Studio 2019
-- Ubuntu

```

Comment:
Make this `image` block look like [this](https://github.com/webmd-health-services/Carbon.Cryptography/blob/main/appveyor.yml#L10-L19).